src/disk_io_thread.cpp

```diff
@@ -1865,7 +1865,7 @@ namespace libtorrent
 	}
 
 	void disk_io_thread::async_check_fastresume(piece_manager* storage
-		, bdecode_node const* resume_data
+		, resume_data_t const* resume_data
```

Comment:
this can probably not be const I'm guessing.. since this is an owning pointer now